AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockIsolatedFilterTest.xml

```diff
@@ -31,10 +31,6 @@
         <actionGroup ref="AdminAdobeStockExpandFiltersActionGroup" stepKey="expandFilters"/>
         <actionGroup ref="AdminAdobeStockApplyIsolatedFilterActionGroup" stepKey="see32imagesOnTheSecondPage"/>
         <grabTextFrom selector="{{AdminAdobeStockSection.recordsFound}}" stepKey="countWithAppliedFilter"/>
-        <assertNotEquals stepKey="checkThatResultCountChangesAfterFilterApplied">
```

Comment:
After closer look to this test I believe that check is necessary, let's revert this change and try to approach it from the modifiers